App/proguard-rules.txt

```diff
@@ -34,9 +34,24 @@
 -dontusemixedcaseclassnames
 -dontskipnonpubliclibraryclasses
 -verbose
+-dontobfuscate
+-forceprocessing
+
+-keep class * extends android.app.Activity
+-assumenosideeffects class android.util.Log {
+    public static *** d(...);
+    public static *** v(...);
+    public static *** i(...);
+    public static *** w(...);
+    public static *** e(...);
+    public static *** wtf(...);
+}
+
+#Building with optimization enabled fails without this.
+-optimizations !code/allocation/variable
 
 # We restrict a few more optimizations for the maps library.
--optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
+#-optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
```

Comment:
I actually thought it was initially commented out. I'll uncomment the line, and in the future I'll inspect git diff more cautiously before commiting. Thanks for reviewing this.